lib/Conversion/TTIRToTTNN/TTIRToTTNN.cpp

```diff
@@ -296,9 +302,10 @@ class SoftmaxOpConversionPattern : public OpConversionPattern<ttir::SoftmaxOp> {
   LogicalResult
   matchAndRewrite(ttir::SoftmaxOp op, OpAdaptor adaptor,
                   ConversionPatternRewriter &rewriter) const override {
+    removeDpsOp(rewriter, adaptor);
```

Comment:
Not necessarily in this change(maybe we can create an issue now), but can we add a paragraph in the doc describing "how to add op" something about DPS vs non-DPS op, and how to handle them in TTNN conversion?

Reply:
Added #872

Reply:
Please associate issue with MLIR Dialect\Passes component, so we don't miss it :)